medusa/providers/torrent/html/bjshare.py

```diff
@@ -81,6 +81,10 @@ def search(self, search_strings, age=0, ep_obj=None, **kwargs):
         if not self.login():
             return results
 
+        manual_search = "manual_search" in kwargs and kwargs['manual_search']
```

Comment:
kwargs.get('manual_search')

Reply:
Actually, i did that to avoid the variable being None, but in this case, it really does not matter... so changed it to simplify the code